test/unit/studio/src/ui/popoverMenus/PopoverManager.test.js

```diff
@@ -112,11 +109,17 @@ Deno.test({
 			const mouseEvent1 = new FakeMouseEvent("click");
 			popover.el.dispatchEvent(mouseEvent1);
 			assertEquals(manager.curtainEl.parentElement, null);
-			assertStrictEquals(manager.current, popover);
+			assertStrictEquals(manager.getLastPopover(), popover);
 
 			// But clicking any other element should
 			const mouseEvent2 = new FakeMouseEvent("click");
-			document.body.dispatchEvent(mouseEvent2);
+
+			try {
```

Comment:
I think this may be an old experiment? I'm not sure what the try catch block is for.